src/libraries/Microsoft.PowerFx.Core/Public/Values/DelegationParameters.cs

```diff
@@ -64,9 +54,43 @@ public virtual IReadOnlyCollection<string> GetColumns()
     [Flags]
     public enum DelegationParameterFeatures
     {
+        // $filter
         Filter = 1 << 0,
+
+        // $top
         Top = 1 << 1,
+
+        // $select
         Columns = 1 << 2,
+
+        // $orderBy
         Sort = 1 << 3,
+
+        // $compute
+        Compute = 1 << 4,
+
+        // $count
+        Count = 1 << 5,
+
+        // $expand
+        Expand = 1 << 6,
+
+        // $format
+        Format = 1 << 7,
+
+        // $index
+        Index = 1 << 8,
+
+        // $levels
+        Levels = 1 << 9,
+
+        // $schemaversion
+        SchemaVersion = 1 << 10,
+
+        // $search
+        Search = 1 << 11,
+
+        // $skip
+        Skip = 1 << 12
```

Comment:
these do not necessarily need to correspond to odata.
We don't need to to add these until we're actually consuming them.